contracts/src/v0.8/l2ep/dev/zksync/ZKSyncSequencerUptimeFeed.sol

```diff
@@ -0,0 +1,272 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import {AggregatorInterface} from "../../../shared/interfaces/AggregatorInterface.sol";
+import {AggregatorV3Interface} from "../../../shared/interfaces/AggregatorV3Interface.sol";
+import {AggregatorV2V3Interface} from "../../../shared/interfaces/AggregatorV2V3Interface.sol";
+import {TypeAndVersionInterface} from "../../../interfaces/TypeAndVersionInterface.sol";
+import {ZKSyncSequencerUptimeFeedInterface} from "./../interfaces/ZKSyncSequencerUptimeFeedInterface.sol";
+import {SimpleReadAccessController} from "../../../shared/access/SimpleReadAccessController.sol";
+import {IL2SharedBridge} from "@zksync/contracts/l2-contracts/contracts/bridge/interfaces/IL2SharedBridge.sol";
+// import {applyL1ToL2Alias} from "@zksync/contracts/l1-contracts/contracts/vendor/AddressAliasHelper.sol";
+
+/**
+ * @title ZKSyncSequencerUptimeFeed - L2 sequencer uptime status aggregator
+ * @notice L2 contract that receives status updates from a specific L1 address,
+ *  records a new answer if the status changed
+ */
+contract ZKSyncSequencerUptimeFeed is
+  AggregatorV2V3Interface,
+  ZKSyncSequencerUptimeFeedInterface,
+  TypeAndVersionInterface,
+  SimpleReadAccessController
+{
+  /// @dev Round info (for uptime history)
+  struct Round {
+    bool status;
+    uint64 startedAt;
+    uint64 updatedAt;
+  }
```

Comment:
Let's add struct packing comments (see style guide [here](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/STYLE_GUIDE.md#structs)):

```suggestion
  struct Round {
    uint64 startedAt; // ─╮ The timestamp at which the round started
    uint64 updatedAt; //  │ The timestamp at which the round was updated
    bool status; // ──────╯ The sequencer status for the round
  }
```

Reply:
is there a difference in the order of the fields here? or we are updating just the comments here?